packages/gasket-plugin-swagger/index.js

```diff
@@ -25,7 +25,13 @@ async function loadSwaggerSpec(root, definitionFile) {
       // eslint-disable-next-line require-atomic-updates
       __swaggerSpec = require('js-yaml').safeLoad(content);
     } else {
-      __swaggerSpec = require(target);
+      fs.access(target, fs.constants.F_OK, (err) => {
```

Comment:
You should probably do this `fs.access` check before trying to read the file as `yaml`, not just if `json`.